Makefile

```diff
@@ -303,10 +303,12 @@ shellcheck:
 	for file in \
 		sbin/safeboot* \
 		sbin/tpm2-attest \
+		sbin/send-to-tpm.sh \
```

Comment:
Is there a reason to name these helpers `.sh`?

Reply:
There is not.  I'll fix it.

Reply:
Fixed.